internal/clients/team3/clientiigo.go

```diff
@@ -153,17 +153,17 @@ func (c *client) ReceiveCommunication(sender shared.ClientID, data map[shared.Co
 				}
 			}
 			// Rule sanctions
-			if _, ok := data[shared.IIGOSanctionScore]; ok {
```

Comment:
I hope this change comes from the main change and you didn't mess with team3 by accident 🤣

Reply:
I think so lol